### PR TITLE
Kevin

### PR DESCRIPTION
**Pull request by FelineHuhn**

### CODE REVIEW

File: index.html
Comment (gunsh0tzz) [inline index.html:1]: Very nice structure! Maybe you could change the headline to something other.

File: index.html [hunk 1]
```diff
@@ -9,6 +9,12 @@
     <header>
       <h1><span>🐉</span>Seeschlangen App<span>🐉</span></h1>
     </header>
+    <main>
+      <article>
+        <h2>Kevin</h2>
+        <p>Content</p>
+      </article>
```
Comment (gunsh0tzz) [inline index.html:16]:
```suggestion
      <section>
        <h2>Kevin</h2>
        <p>Content</p>
      </section>
```

**Review by gunsh0tzz — APPROVED** [review]
Great progress!

File: index.html
Comment (FelineHuhn) [reply index.html:1]: Thank you, but I prefer this version :)

**Pull request merged**